core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/identity/JWTSupport.java

```diff
@@ -278,17 +278,13 @@ public static User toUser(DecodedJWT jwt, JWTConfiguration config) {
 			userId = emailClaim.asString();
 		}
 
-		final List<String> permissions;
+		List<String> permissions = Collections.emptyList();
 		final String permissionsClaimProperty = config.getPermissionsClaimProperty();
 		if (!Strings.isNullOrEmpty(permissionsClaimProperty)) {
 			Claim permissionsClaim = jwt.getClaim(permissionsClaimProperty);
-			if (permissionsClaim == null || permissionsClaim.isNull()) {
-				throw new BadRequestException("'%s' JWT access token field is required for permissions access, but it was missing.",
-						permissionsClaimProperty);
+			if (permissionsClaim != null && !permissionsClaim.isNull() && !permissionsClaim.isMissing()) {
+				permissions = permissionsClaim.asList(String.class);
 			}
-			permissions = permissionsClaim.asList(String.class);
-		} else {
-			permissions = Collections.emptyList();
 		}
 
 		return new User(userId, permissions.stream().map(Permission::valueOf).collect(Collectors.toList()), jwt.getToken());
```

Comment:
Does an empty collection mean "not permitted to do anything" or "unknown permissions"? It can be that there is no practical distinction between the two, but e.g. an `anyMatch` call returns `true` for an empty `Stream` so we might need to add extra checks elsewhere.